main.go

```diff
@@ -81,11 +85,29 @@ func main() {
 		os.Exit(1)
 	}
 
+	hostPosts := strings.Split(hostPortRange, ",")
+	portStart, err := strconv.Atoi(hostPosts[0])
+	if err != nil {
+		setupLog.Error(err, "port should have int type")
+		os.Exit(1)
+	}
+
+	portEnd, err := strconv.Atoi(hostPosts[1])
+	if err != nil {
+		setupLog.Error(err, "port should have int type")
+		os.Exit(1)
+	}
+
```

Comment:
validate hostPosts does have a length of 2? Or hostPosts[1] may crash if the input is ill-formed.

Reply:
whatever, anyway the program needs to exit